### ISSUE TITLE
[BUG] Pending uploads not visible in admin panel when in single gallery mode

### ISSUE THREAD

**Issue by justindixson** [Cirx08/WeddingShare#15]
When using the single gallery mode environment variable, pending photos do not show in the admin panel when multiple folders exist within the mount path. It appears to pick up any folder within the mounted volume as a gallery which may be causing it some confusion when looking for pending approvals.

Workaround for now is to keep the mounted volume 'clean' with only the default gallery folder which is likely intended but could be a but more robust in the future.

Side note, love the project so far!

## Example compose file (with specifics removed)

```docker-compose
services:
  wedding-share:
    container_name: WeddingShare
    image: cirx08/wedding_share:latest
    restart: unless-stopped
    profiles: ["web", "all"]
    ports:
      - '${HTTP_PORT:-5000}:5000/tcp'
    environment:
      TITLE: 'Wedding Share'
      LOGO: 'logo'
      GALLERY_COLUMNS: 4
      ALLOWED_FILE_TYPES: '.jpg,.jpeg,.png'
      SINGLE_GALLERY_MODE: 'true'
      DISABLE_HOME_LINK: 'true'
      MAX_FILE_SIZE_MB: 20
      IDLE_GALLERY_REFRESH_MINS: 2
      REQUIRE_REVIEW: 'true'
      # SECRET_KEY: 'password'
      ADMIN_PASSWORD: 'password'
    volumes:
      - /mnt/Volume/uploads:/app/wwwroot/uploads
    hostname: wedding-share

```

## Example uploads folder structure (shared uploads folder)

```
uploads/
uploads/default/
uploads/config/
uploads/.stfolder/
```

## Screenshots

Gallery with pending reviews

![Image 11-30-24 at 9 08 PM](https://github.com/user-attachments/assets/d2530908-6083-4750-89ab-0020cf0a868e)

Admin Panel

<img width="1359" alt="SCR-20241130-sinr" src="https://github.com/user-attachments/assets/999fad11-efe3-4734-9da3-2024e6ed1e70">

**Comment by Cirx08** [comment]
Can you please clarify a few things for me because I haven't been able to reproduce the issue. Are the pending images in either the Config or .stfolder directories? Is there a .stfolder also inside the above directories? Does it happen when you aren't using SyncThing?

**Comment by justindixson** [comment]
Hi thanks for the quick reply! Sure thing, the pending images are only in the "default" folder created on first start while in single gallery mode, .stfolder exists only at the root, not within the gallery folder and is an empty stub.
The config folder has the following files created by Syncthing, likely of no consequence to this issue but just in case:

```
cert.pem 
config.xml.v0
https-key.pem
key.pem
config.xml
https-cert.pem
index-v0.14.0.db
```

Once both folders were removed from the volume mounted to wedding-share the pending image will appear.

As a test on my own setup with a new instance and using the same setting above, I used a clean (empty) uploads directory then uploaded a file and approved without issue (still in single gallery mode). 
Then I uploaded a second photo and left it in the queue, it will show in the admin panel. I then created an empty folder (I'll call it folder2 for this example) in the uploads directory next to the "default" folder that gets created while in single gallery mode, checked the admin panel, and the pending item was gone with a second "gallery" of the new empty folder. I then deleted the empty "folder2", refreshed the admin panel and the pending photo appears again.
This test was without Syncthing and just playing with creating and deleting empty folders.

Hope this helps and thanks again!
